source/common/http/http2/codec_impl.h

```diff
@@ -176,6 +176,18 @@ class ConnectionImpl : public virtual Connection, protected Logger::Loggable<Log
     const Network::Address::InstanceConstSharedPtr& connectionLocalAddress() override {
       return parent_.connection_.localAddress();
     }
+    absl::string_view responseDetails() override { return details_; }
+
+    void setDetails(absl::string_view details) {
+      // Figure out whether we can guarantee that details_ is set at
```

Comment:
Again if you're not going to follow up on this I think you can simply say
// It is probably a mistake to call setDetails() twice, so assert that details_ is empty.

Reply:
ack, will be in the next push